src/openrct2-ui/windows/TrackDesignPlace.cpp

```diff
@@ -77,6 +77,8 @@ namespace OpenRCT2::Ui::Windows
     );
     // clang-format on
 
+    static bool gPlacingTrackDesign = false;
```

Comment:
We typically only use the `g` prefix if the variable is exposed through headers, fwiw.